### ISSUE TITLE
Please add a changelog and 2.0.0 tag

### ISSUE THREAD

**Issue by tomek-he-him** [jouni-kantola/inline-chunk-manifest-html-webpack-plugin#20]
Hey, thanks for this nice package! I’m in a bit of a crunch though, because I need to update from v1.1.2 to v2.0.0 due to a problem with chunk-manifest-webpack-plugin peer dependencies. Can’t figure out what changed though.

Judging by the semver bump there was a breaking change. What was it?

Two things would help greatly: adding a changelog (you can find some good tips at http://keepachangelog.com/en/1.0.0/) and tagging the commits at which you published the package.

Thanks again!

**Comment by jouni-kantola** [comment]
Good that you reminded me about the changelog, thank you. I usally do, forgot this time for some reason. For now I'll let you know the changes here in the issue:

*tl;dr*
- Forked `chunk-manifest-webpack-plugin` and removed overlaps
- Major bump because removed webpack 1 support
- API wise no updates

*Long version*
What was changed is that I forked `chunk-manifest-webpack-plugin`, so it's now a part of this plugin. I want to be able to make changes independently. I removed the bits that were complete overlaps, and made sure the chunk manifest wasn't written to chunks anymore (which was the reason why I never moved to `chunk-manifest-webpack-plugin` v1.1.2, issue: https://github.com/soundcloud/chunk-manifest-webpack-plugin/issues/47). 
The reason for the major bump was because the `inline-chunk-manifest-html-webpack-plugin` doesn't support webpack 1 anymore. I can't recall ever testing webpack 1 in the first place anyways. With the major bump, for you as a consumer of the plugin, nothing's changed regarding the configuration of the plugin (unless you already before passed in `chunk-manifest-webpack-plugin`, which was never required).
